core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -551,7 +551,7 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
   public static final SqlFunction REGEXP_SUBSTR = REGEXP_EXTRACT.withName("REGEXP_SUBSTR");
 
   /** The "REGEXP(value, regexp)" function, equivalent to {@link #RLIKE}. */
-  @LibraryOperator(libraries = {SPARK})
+  @LibraryOperator(libraries = {SPARK, HIVE})
```

Comment:
According to the official documentation, Hive does not support REGEXP(A, B).

https://cwiki.apache.org/confluence/display/Hive/LanguageManual+UDF
<img width="333" alt="image" src="https://github.com/apache/calcite/assets/7956306/2cf95634-a354-45e1-a2eb-bb4e8dd08f98">

Reply:
Yes, I try A REGEXP B, not the REGEXP(A, B). Thanks for the review, I will rollback the code.